cluster/bin/snappy-job.sh

```diff
@@ -119,6 +120,15 @@ while (( "$#" )); do
       shift
       contextName="${1:-$TOK_EMPTY}"
     ;;
+    --user)
+      shift
+      username="${1:-$TOK_EMPTY}"
+      while [ -z ${password} ]; do
+        read -s -p "Enter password for ${username}: " password
+        echo ""
+      done
+      securePart="-u ${username}:${password}"
+    ;;
```

Comment:
Yes, switched to --netrc before merging the change into master (https://github.com/SnappyDataInc/snappydata/pull/731/commits/09c66ff9060f4d5ea85eab1f0013d11d73ab0278)
But later saw some issues with it so changed to using -K or --config option (https://github.com/SnappyDataInc/snappydata/commit/b5f0fde11008b65c28415b9590c923077940c522). This one is yet to be merged into master.